### DISCUSSION TITLE
Disable Randomizing Kingdoms - Junior Warlord Stories

### DISCUSSION THREAD

**Discussion by AdmiralSwampert** [Deijin27/RanseiLink#63] [Q&A]
Is there a way to disable the randomization of kingdoms in Junior Warlord Stories?

**Comment by Deijin27** [comment]
If you mean the "Warrior's Pokemon" setting, which randomises which pokemon are assigned to which scenario-warrior slot. Then no.

It would be helpful if you could clarify what you mean, and explain why it would be useful.
Once I understand, I can consider how to implement the feature.

**Reply by Baconfry** [reply]
In Junior Warlord stories, each army controls a random subset of the available kingdoms. The number of kingdoms depends on the army leader, and the kingdoms have to border each other, but otherwise it appears to be random. I think this is what they were asking.
